package.json

```diff
@@ -70,7 +70,7 @@
     "@bbc/psammead-inline-link": "^1.3.11",
     "@bbc/psammead-locales": "^3.0.0",
     "@bbc/psammead-media-indicator": "^2.6.4",
-    "@bbc/psammead-media-player": "^2.0.0",
+    "@bbc/psammead-media-player": "^2.1.0",
```

Comment:
Just a heads up that today I discovered `psammead-media-player@2.1.0` is broken. `AmpImg` within `AmpMediaPlayer` is not being passed the required `width` and `height` and needs to be fixed upstream.

Here's a screenshot of one of the test failures from within Simorgh.
<img width="785" alt="Screenshot 2019-10-29 at 22 09 58" src="https://user-images.githubusercontent.com/1430787/67813398-57b60e00-fa99-11e9-845b-5c10bc69a0f5.png">

It looks like a fix is being worked on – https://github.com/bbc/psammead/pull/2516. Cc @jamesdonoh